phpBB/includes/acp/acp_board.php

```diff
@@ -450,6 +450,7 @@ function main($id, $mode)
 						'email_enable'			=> array('lang' => 'ENABLE_EMAIL',			'validate' => 'bool',	'type' => 'radio:enabled_disabled', 'explain' => true),
 						'board_email_form'		=> array('lang' => 'BOARD_EMAIL_FORM',		'validate' => 'bool',	'type' => 'radio:enabled_disabled', 'explain' => true),
 						'email_package_size'	=> array('lang' => 'EMAIL_PACKAGE_SIZE',	'validate' => 'int:0',	'type' => 'number:0:99999', 'explain' => true),
+						'email_max_chunk_size'	=> array('lang' => 'EMAIL_MAX_CHUNK_SIZE',	'validate' => 'int:0',	'type' => 'number:1:99999', 'explain' => true),
```

Comment:
You validate the number with `min = 0`, yet the `type` (which creates the HTML) has a `min = 1 && max = 99999`. Seems inconsistent.

Reply:
Thx m8 :+1: